tests/libnet/cluster.go

```diff
@@ -0,0 +1,38 @@
+package libnet
+
+import (
+	"fmt"
+
+	k8sv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	netutils "k8s.io/utils/net"
+
+	"kubevirt.io/client-go/kubecli"
+)
+
+func IsClusterDualStack(virtClient kubecli.KubevirtClient) bool {
+	// grab us some neat kubevirt pod; let's say virt-handler is our target.
+	targetPodType := "virt-handler"
+	virtHandlerPod, err := getPodByKubeVirtRole(virtClient, targetPodType)
+	if err != nil {
+		return false
+	}
+	for _, ip := range virtHandlerPod.Status.PodIPs {
+		if netutils.IsIPv6String(ip.IP) {
+			return true
+		}
+	}
+	return false
+}
+
+func getPodByKubeVirtRole(virtClient kubecli.KubevirtClient, kubevirtPodRole string) (*k8sv1.Pod, error) {
+	labelSelectorValue := fmt.Sprintf("%s = %s", "kubevirt.io", kubevirtPodRole)
+	pods, err := virtClient.CoreV1().Pods("kubevirt").List(metav1.ListOptions{LabelSelector: labelSelectorValue})
```

Comment:
This will not work if we run with HCO or CNV since namespace is not "kubevirt", there is a generic way to do this that I don't remember.